src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -64,6 +64,24 @@ public function chooseDifferentShippingAddress(): void
         $billingAddressSwitch->check();
     }
 
+    public function chooseDifferentBillingAddress(): void
+    {
+        $driver = $this->getDriver();
+        if ($driver instanceof Selenium2Driver || $driver instanceof ChromeDriver) {
+            $this->getElement('different_billing_address_label')->click();
+
+            return;
+        }
+
+        $billingAddressSwitch = $this->getElement('different_billing_address');
+        Assert::false(
+            $billingAddressSwitch->isChecked(),
+            'Previous state of different billing address switch was true expected to be false',
+        );
+
+        $billingAddressSwitch->check();
```

Comment:
This whole content is quite similar to the previous function, could be extracted with some `string $addressType` parameter